build.sh

```diff
@@ -0,0 +1,179 @@
+#! /bin/bash
```

Comment:
yes, we already have a dependency on git, and the build script is not really needed for day to day dev, it mostly just runs on the build servers.